frontend/src/concepts/explainability/content/TrustyAIServiceNotification.tsx

```diff
@@ -0,0 +1,66 @@
+import { Alert, AlertActionCloseButton, Bullseye, Spinner } from '@patternfly/react-core';
+import React from 'react';
+import TrustyAITimedOutError from '~/concepts/explainability/content/TrustyAIServerTimedOutError';
+
+type TrustyAIServiceNotificationProps = {
+  error?: Error;
+  isAvailable: boolean;
+  showSuccess: boolean;
+  loading: boolean;
+  timedOut: boolean;
+  ignoreTimedOut: () => void;
+  deleteCR: () => Promise<unknown>;
+};
+
+const TrustyAIServiceNotification: React.FC<TrustyAIServiceNotificationProps> = ({
+  error,
+  isAvailable,
+  showSuccess,
+  loading,
+  timedOut,
+  ignoreTimedOut,
+  deleteCR,
+}) => {
+  const [dismissSuccess, setDismissSuccess] = React.useState(false);
+
+  if (timedOut) {
+    return <TrustyAITimedOutError ignoreTimedOut={ignoreTimedOut} deleteCR={deleteCR} />;
+  }
+
+  if (loading) {
+    if (dismissSuccess) {
+      setDismissSuccess(false);
```

Comment:
Maybe just nix this whole logic -- what are you protecting?

Reply:
Ok I threw this over the fence at the end of the day hah, I should have seen this, ouch that's bad! :D 

This is basically because I need to reset the dismiss success flag, otherwise if you install, then uninstall, then install again you won't get the success notification. Let me fix that up.